config/config.yaml.dist

```diff
@@ -54,39 +51,40 @@ database:
     port: 3306
     # tls: "disable"
     tls: "false"
-    user: "sparky"
-    role: ""
-    password: "sparky123"
+    user: ""
+    # role: ""
+    password: ""
     name: "pipeline"
 
     # Turn this to true to see all queries in the log
-    queryLog: false
+#    queryLog: false
 
-    autoMigrate: true
+#    autoMigrate: false
 
 cadence:
-    host: "127.0.0.1"
+    host: ""
 #    port: 7933
 #    domain: "pipeline"
-    createNonexistentDomain: true
+#    createNonexistentDomain: false
 #    workflowExecutionRetentionPeriodInDays: 3
 
-cors:
-    allowAllOrigins: false
-    allowOrigins: ["http://localhost:4200"]
+#cors:
+#    allowAllOrigins: true
```

Comment:
Please, add a note about the risks of running even a development instance with this option left true.

Reply:
Thanks!